tool/tsh/proxy.go

```diff
@@ -275,6 +276,7 @@ func mkLocalProxy(ctx context.Context, opts localProxyOpts) (*alpnproxy.LocalPro
 		ParentContext:      ctx,
 		SNI:                address.Host(),
 		Certs:              certs,
+		ExtraProtos:        opts.extraProtos,
```

Comment:
Can we just change the Protocol to a  array instead of adding new field ?

Reply:
Done